content/member-handbook.md

```diff
@@ -147,6 +147,186 @@ That said, here are some things we might do, in rough order of severity:
+    company. "Unaffiliated" is a valid option, however any recruiters found to
+    not have disclosed that they are recruiters when posting jobs will be
+    permenantly banned.
+  - The Type of Position: All postings MUST disclose whether they are: Direct
+    hire, contract, or contract to hire.
+  - Employer or staffing company name: All job postings MUST include the name of
+    the company that the candidate would be directly working for, OR, the name
+    of the agency that the recruiter works for.  Posting the name of the
+    employer is not required, but is HIGHLY ENCOURAGED.  If a recruiter posts a
+    job without the employer name, and another member of the community discloses
+    the name of the employer, the moderation team will not take any action.
+  - Adhere to the CoC: All job postings must adhere to the STL Tech community
+    standards and code of conduct.
+
+##### Highly Encouraged Information
+
+We suggest that job postings include as much of the following information as
+possible.  While job postings without this information are not outright
+prohibited, users who continually post jobs without any of the following
+information may be asked on a case-by-case basis to either include more
+information in their postings or to stop posting.
+
+  - Employer Name:  As stated above, the employer name is not required, however
+    it is highly encouraged that all job postings SHOULD include the name of the
+    employer where the candidate would be working.
+  - Salary: Job postings SHOULD include a reasonable salary range.  It's better
+    to guess at this than to omit it, unless you really have no idea at all.
+  - Contract Margins:  For contract and contract-to-hire positions, recruiters
+    SHOULD include their hourly or fixed cost margins that are added to the
+    candidates salary.
+  - Benefits: Job postings SHOULD include a list of available benefits, such as
+    amount of PTO, health insurance, paternity or maternity leave, educational
+    benefits, etc.
+  - Number of positions: A job posting SHOULD disclose how many candidates are
+    expected to be hired for the given position.
+  - Type of work: Many job postings include a list of technology skills and
+    requirements, but do not speak to the nature of the work.  While not
+    required, job postings SHOULD include a description of the type of work to
+    be performed.
+
+##### Additional Considerations
+
+Job postings on the STL Tech Slack should be crafted to speak to our community.
+There is no prohibition against merely copying information from an existing job
+posting, but we encourage members of the STL Tech community go personalize the
+message for the audience.  In particular:
+
+  - Leverage tools like [joblint](http://joblint.org) to ensure your job posting
+    does not contain language that would exclude marginalized people.
+  - Avoid terms like "rockstar" and "ninja".
```

Comment:
this is covered in joblint, isn't it?

Reply:
It does, but further iteration, so people are totally clear. Plus Joblint sometimes misses some things you'd really expect to get covered.